iac-extensions/terraform/src/test/resources/checks/Azure/RoleBasedAccessControlCheck/kubernetes_cluster_version3.tf

```diff
@@ -31,5 +31,25 @@ resource "azurerm_kubernetes_cluster" "role_based_access_control_enabled" {
   role_based_access_control_enabled = true
 }
 
+resource "azurerm_kubernetes_cluster" "noncompliant_azure_active_directory_role_based_access_control" {
+  azure_active_directory_role_based_access_control {
+    managed = true
+    azure_rbac_enabled = false # Noncompliant {{Make sure that disabling role-based access control is safe here.}}
+  }
+}
+
+resource "azurerm_kubernetes_cluster" "incomplete_azure_active_directory_role_based_access_control" {
+  azure_active_directory_role_based_access_control {
+    managed = true # Noncompliant {{Omitting 'azure_rbac_enabled' disables role-based access control for this resource. Make sure it is safe here.}}
```

Comment:
I guess that you managed to find what's the default value for `azure_active_directory_role_based_access_control.azure_rbac_enabled`, I don't see it on the [azurerm_kubernetes_cluster](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/kubernetes_cluster) documentation.

Reply:
I checked the [implementation](https://github.com/hashicorp/terraform-provider-azurerm/blob/29e5835f814827186d0e943562926357c93227a0/internal/services/containers/kubernetes_cluster_data_source.go#L1077) and IMO the default value seems to be false in any case.